traffic_monitor/towrap/towrap.go

```diff
@@ -680,20 +680,22 @@ func (s TrafficOpsSessionThreadsafe) MonitorCDN(hostName string) (string, error)
 // Traffic Monitors and Delivery Services found in a CDN Snapshot, and wipe out
```

Comment:
I believe we can remove the "[...] and wipe out all of those..." phrase as well since we will no longer be doing that.